src/sentry/static/sentry/app/components/charts/eventsRequest.tsx

```diff
@@ -236,9 +242,13 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
   private unmounting: boolean = false;
 
   fetchData = async () => {
-    const {api, ...props} = this.props;
+    const {api, confirmedQuery, ...props} = this.props;
     let timeseriesData: EventsStats | MultiSeriesEventsStats | null = null;
 
+    if (confirmedQuery === false) {
```

Comment:
Can we add a comment here? (Or maybe in the type def) I think we should clarify that we do a strict equality check for `false` because the default behavior is to not wait for confirmation (e.g. when it's undefined).